src/main/scala/rpsGame.scala

```diff
@@ -1,69 +1,65 @@
 object rpsGame {
-def main(args: Array[String]):Unit = {
+  var pRCounter: Int = 0
+  var pPCounter: Int = 0
+  var pSCounter: Int = 0
+  var pWin: Int = 0
+  var aWin: Int = 0
+  var firstMove: Boolean = true
   var quit: Boolean = true
```

Comment:
You only use this in line 14 and 15, never resigning the value to it, by the looks of it you run the game 4ever